dashboards/VirtAPIDown.md

```diff
@@ -0,0 +1,25 @@
+# VirtAPIDown
```

Comment:
@sradco  should not this be under `runbooks` folder?

Reply:
@erkanerol Yes. Fixed. Thank you.